core/domain/user_services_test.py

```diff
@@ -14,6 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+from contextlib import nested
```

Comment:
Also, just for future reference: we always import the top-level library, rather than classes/functions within it. So this would have been "import contextlib", and the code would use `contextlib.nested(...)`.